anda/apps/helium-browser/helium-browser.spec

```
@@ -0,0 +1,115 @@
+%define debug_package %{nil}
+%global _build_id_links none
+
+%global __requires_exclude libffmpeg.so|libvk_swiftshader.so|libvulkan.so|libEGL.so|libGLESv2.so
+%global __provides_exclude_from /opt/%{name}/.*\\.so
+
+Name:           helium-browser
```

```suggestion
Name:           helium-browser-bin
```
might wanna do this to specify that it's their binary